interface-elements-for-web/articles/grid/customization-dialog/filtering-page.md

```diff
@@ -4,6 +4,24 @@ author: Anna Kondratova
 legacyId: 118822
 ---
 # Filtering Page
-The "Filtering" page allows you to create and change the filter conditions. For each column, select the criteria operator and specify the operand value.
+The **Filtering** page allows you to create and change filter conditions. 
+
+For each column, you can select the criteria operator and specify the operand value.
+
+* Use ![eud-grid-customizationdialog-close-button](../../../images/grid-customizationdialog-expandfilter.png) to expand a column filter condition.
+* Use ![eud-grid-customizationdialog-close-button](../../../images/grid-customizationdialog-removefilter.png) or clear an operand value, to remove a column filter condition.
+
+![eud-grid-customizationdialog-filteringpage](../../../images/img128925.png)
+
+Click a column criteria operator to change it. Select the new operator from the displayed menu.
+
+![eud-grid-customizationdialog-filteringpage](../../../images/grid-cust-dialog-filtering.png)
+
+
+
+Click the **Apply** (![eud-grid-customizationdialog-apply-button](../../../images/grid-customizationdialog-apply.png)) or **Close** (![eud-grid-customizationdialog-close-button](../../../images/grid-customizationdialog-close.png)) button to apply or discard the changes, respectively.
```

Comment:
Same changes

Reply:
Perhaps (without "on every page", because the topic describes only one dialog page):
"You can modify dialog settings and click Apply [img] or click Close [img] to discard all the changes."